README.md

```diff
@@ -11,6 +11,16 @@ The `TestRun` CRD is a representation of a single k6 test executed once. `TestRu
 
 The `PrivateLoadZone` CRD is a representation of a [load zone](https://grafana.com/docs/grafana-cloud/testing/k6/author-run/use-load-zones/), which is a k6 term for a set of nodes within a cluster designated to execute k6 test runs. `PrivateLoadZone` is integrated with [Grafana Cloud k6](https://grafana.com/products/cloud/k6/) and requires a [Grafana Cloud account](https://grafana.com/auth/sign-up/create-user). You can find a guide describing how to set up a `PrivateLoadZone` [here](https://grafana.com/docs/grafana-cloud/testing/k6/author-run/private-load-zone-v2/), while billing details can be found [here](https://grafana.com/docs/grafana-cloud/cost-management-and-billing/understand-your-invoice/k6-invoice/).
 
+## Installation
+
+To install the k6 Operator, use the following command:
+
+```bash
+curl https://raw.githubusercontent.com/grafana/k6-operator/main/bundle.yaml | kubectl apply -f -
+```
+
+For more information, see the [Install k6 Operator](https://grafana.com/docs/k6/latest/set-up/set-up-distributed-k6/install-k6-operator/) page.
+
 ## Documentation
 
 You can find the latest k6 Operator documentation in the [Grafana k6 OSS docs](https://grafana.com/docs/k6/latest/set-up/set-up-distributed-k6/usage/common-options/).
```

Comment:
Specifically, installation instructions for the k6 Operator can be found on [this page](https://grafana.com/docs/k6/latest/set-up/set-up-distributed-k6/install-k6-operator/).

Reply:
Either works for me. I just couldn't find what I was looking for by looking at the README and clicking through several times. The installation page is kinda hidden by the linked pages, and one must click a few times to find it.